ApollosProject/apollos-embeds
# feat: 📊 Web analytics

## Scope

[🐤 Big Board Ticket](https://3.basecamp.com/3926363/buckets/27088350/card_tables/cards/6655727656)

[📊 Pitch](https://whimsical.com/web-analytics-M5UD9eE2uofDVSSRjjeaVQ)

[📊 Todo List](https://3.basecamp.com/3926363/buckets/27088350/todolists/6819968423)

Added amplitude and segment to web embeds

## 🔬 To Test

Need to add `REACT_APP_APOLLOS_SEGMENT_KEY` to `.env`. I think you can find the key on the config db if you query for `APP.SEGMENT_KEY`

1. Run web embeds with church of choice (apollos_demo and liquid are probably best)
2. Open up amplitude and segment and ensure correct things are being tracked
3. Navigate to content singles and login in and out. Should see that activity reflected in amplitude
4. Navigate to a feature feed to see event logged
5. Navigate to a video and play around with the video settings. Events should be logged
6. should still work the same
7. Make sure trackEvent names are correct and in the right place


https://github.com/ApollosProject/apollos-embeds/assets/68402088/d39a3b65-e833-4f14-836c-b4a415c77612



## 📸 Screenshots

<img width="1478" alt="image" src="https://github.com/ApollosProject/apollos-embeds/assets/68402088/03326f85-e978-48a4-919e-797523411302">
